### PR TITLE
Updated toastify version to 1.8.3

### PR DESCRIPTION
**Pull request by Grahamvs**
Updated Toastify version to 1.8.3

### CODE REVIEW

File: toastify/toastify.nuspec [hunk 1]
```diff
@@ -4,9 +4,9 @@
   <metadata>
    <id>toastify</id>
    <title>toastify</title>
-   <version>1.7.8</version>
+   <version>1.8.3</version>
```
Comment (ghuntley) [inline toastify/toastify.nuspec:7]: :+1:

File: toastify/toastify.nuspec [hunk 2]
```diff
@@ -4,9 +4,9 @@
...
    <authors>Geoffrey Huntley</authors>
-   <owners>ghuntley</owners>
+   <owners>ghuntley, slayerice09</owners>
```
Comment (ghuntley) [inline toastify/toastify.nuspec:9]: :-1:

File: toastify/toastify.nuspec [hunk 3]
```diff
@@ -19,6 +19,20 @@
      <dependency id="spotify" version="0.9.8.296" />
    </dependencies>
    <releaseNotes>
+    # Toastify 1.8.3 - 10/3/2016
```
Comment (ghuntley) [inline toastify/toastify.nuspec:22]: :+1:

File: toastify/tools/chocolateyInstall.ps1 [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ﻿$packageName = 'toastify'
 $installerType = 'EXE' 
-$url = 'https://toastify.codeplex.com/downloads/get/1480805' 
-$url64 = 'https://toastify.codeplex.com/downloads/get/1480805'
+$url = 'https://toastify.codeplex.com/downloads/get/1554919'
```
Comment (ghuntley) [inline toastify/tools/chocolateyInstall.ps1:3]: :+1:

File: toastify/toastify.nuspec (hunk 2 above)
Comment (Grahamvs) [inline toastify/toastify.nuspec:9]: What's the problem with this? I am now one of the maintainers for toastify on chocolatey as they threatened to remove it if it didn't...

Comment (ghuntley) [inline toastify/toastify.nuspec:9]: Wait.. How did you become a maintainer? Chocolatey didn't run this by me.

Comment (Grahamvs) [inline toastify/toastify.nuspec:9]: A while ago I send a message through the site to the "maintainer" (guessing you?) saying there was an update, but didn't hear or see anything, so yesterday (following the site's rules) I used the "contact the site admin" saying there was a new update. They replied saying either I become a maintainer or they remove it... If you're still maintaining it, I'm happy to back off...

Comment (Grahamvs) [inline toastify/toastify.nuspec:9]: Where do we stand with this? Are you still maintaining this? If so, I'll remove my name from the file and let you update it. If not, I'm happy to take over maintaining toastify...

Comment (ghuntley) [inline toastify/toastify.nuspec:9]: Alright, I've spoken with one of the chocolatey maintainers - they didn't follow process properly and added you as a maintainer without contacting myself which caught me off-guard. My qualm is with Chocolatey, not you. Happy to have you as a maintainer.

**Pull request merged**